thesis/experience.tex

```diff
@@ -0,0 +1,25 @@
+\chapter{Experience}
+In the process of developing the applications and working with the domain we have made some valuable experiences.
+
+Overall, the system engineering challenge of grasping the complete domain, breaking it down into consumable parts and understanding the needs of each of these is a repeatable challenging task.
+We have made use of the methods used by modern software development teams and combined it with the needs of this thesis' educational requirements and this has proven very useful.
+
+The \glsi{scrum} methodology have given us an incredible insight into how we estimate and define tasks, while keeping the overall goal in sight.
+Combined with the version control software this has given us the ability to work with the same flows and terms as if it was a professional development team with release notes, pull requests, reviews and more.
+
+We have used user surveys and interviews to understand the requirements of the products and used this to prioritize what tasks to do first.
+Further more, we have used the same consultants to look into application flow mock ups and direct us to do make the right choices.
+
+An important lesson that we learned was to be more specific when we set requirements to our collaborators.
+The hardware for the rackets was not completed in time and simple misunderstandings of what the minimum requirements were, have delayed the process of developing the sensors.
+From this, we now know that we must be as specific with our own requirements to collaborators as we are when setting up requirements for our own software.
+
+All these things combined have learned us how important it is to have a transparent and understandable work flow as well as clear requirements to what needs to be done.
+
+On the more technical parts, we have learned that sales sites tend to show a bright picture of what a framework can accomplish.
+We were sure \glsi{nativescript} was the right choice for us when ending the pre-study on cross-platform frameworks, section \ref{sec:crossplatform}, but when getting into the development phase of things, it turned out to be much more cumbersome and tedious to get good results.
+We were not able to set up testing on device emulators and had problems with getting the included tooling like \textit{hot reload} of the application to work.
+As testing is an important part of improving and maintaining software quality, we set up a stand alone testing environment. that made functional tests of the code.
+
+On the other hand, have we only been positively impressed with how clear and straight to the point the \glsi{graphql} specification allows for development, and how the \glsi{apollo} framework enables high productivity when writing the software.
```

Comment:
on the other hand, have we only been

shouldnt it be more like:

on the other have, we have only been ??